int8 model
 
---
int32 result